.github/workflows/python-main.yml

```
@@ -159,13 +159,13 @@ jobs:
         run: echo Home directory inside container $HOME
 
       - name: Cache vcpkg folder
-        if: matrix.os != 'windows-2022'
+        if: runner.os != 'Windows'
         uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)